### PR TITLE
Ps 114 hide login and register in navbar

### PR DESCRIPTION
**Pull request by JafariM**

### CODE REVIEW

File: src/components/SignIn.js
Comment (akosasante) [inline src/components/SignIn.js:1]: I'm not sure if these changes are meant to be in this PR.

File: src/util/mirageServer.js [hunk 1]
```diff
@@ -15,6 +15,38 @@ export function makeServer() {
       // <-- FIX
 
       this.get('http://localhost:8000/api/v1', { data: 'This is a music app' }),
+        // route to submit user registration form
```
Comment (akosasante) [inline src/util/mirageServer.js:18]: Maybe add a little comment to the top of this file directing developers to uncomment any routes that they need/want to mock

**Review by akosasante — APPROVED** [review]
Navbar and mirageServer changes look good. Probably don't want the SignIn changes here since they're being reviewed in a separate PR.

File: src/util/mirageServer.js (hunk 1 above)
Comment (JafariM) [reply src/util/mirageServer.js:18]: I will add the comment

**Pull request merged**